components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/executors/ApiPublisherExecutor.java

```diff
@@ -110,6 +105,7 @@ private static APIStatus getApiStatus(String status) {
         for (APIStatus aStatus : APIStatus.values()) {
```

Comment:
We remove this logic from the code 
https://github.com/wso2/carbon-apimgt/pull/577

Thanks